manual/src/main/asciidoc/migrations/migrate-1.6-to-2.0.adoc

```diff
@@ -38,8 +38,19 @@ Once you updated your applications to align with Unomi 2 data model, the next st
 Any event (and more generally, any object) received through Unomi public endpoints do require a valid JSON schema.
 Apache Unomi ships, out of the box, with all of the necessary JSON Schemas for its own operation as well as all event types generated from the Apache Unomi Web Tracker but you will need to create schemas for any custom event you may be using.
 
-When creating your new schemas, you can setup debug messages in the logs (using: `log:set DEBUG org.apache.unomi.schema.impl.SchemaServiceImpl` in Karaf console) that
-will point to errors in your schemas or will help you diagnose why the events are not being accepted. It is also possible to use the UNOMI_LOGS_JSONSCHEMA_LEVEL environment variable (by setting it to the `DEBUG` value) and then restarting Apache Unomi to accomplish the same thing. The second option is especially useful when using Docker containers. It is also possible to test if your events are valid with the a new API endpoint mapped at `/cxs/jsonSchema/validateEvent`.
+When creating your new schemas, there are multiple ways of testing them:
+
+- Using a the event validation API endpoint mapped at : `/cxs/jsonSchema/validateEvent`
+- Using debug logs when sending events using the usual ways (using the `/context.json` or `/eventcollector` endpoints)
+
+Note that in both cases it helps to activate the debug logs.
```

Comment:
`...helps to activate the debug logs, you can activate them either:`

Reply:
Ok I'll change that